Worksheet.md

```diff
@@ -134,15 +136,22 @@ This takes the **output** of `grep` and _pipes_ it to the **input** of the `wc`
 
 ### System administration
 
-There are many tools you can use in Linux to keep on top of your computer.
+There are many tools you can use in Linux to keep on top of your computer, as well as some useful shortcuts and tricks. 
 
-The first tool we'll look at is `man` - these are "manual pages" (also referred to as manpages), and they provide explanation of the commands in Linux and all of the different things you can do with them. Once you're looking at a manpage, you can use the arrow keys on your keyboard to scroll through the document, and press the `Q` key on your keyboard to exit.
+- `Ctrl + C` is a keyboard interrupt, and stops whatever process is being run on your terminal window. 
+- Adding `&` to the end of a command in a terminal will make it run in the background, letting you do other things while it's running. (You can also use `command1 && command2` if you want to run two commands one after the other). 
+- Copying and pasting in terminal windows uses `Ctrl + Shift + C/V` rather than just `Ctrl + C/V`. 
+- `clear` will clear the terminal window of all previous text. 
+- You can use the Tab key for the terminal to attempt to autocomplete your command or the directory you want to move to.
 
-Another tool that's useful is `ps` - this is a tool that shows us all of the running programs (called processes) on our computer. Typing `ps -e` will show all running programs on the computer in a list. You can also use `pstree` to show the process tree - processes in Linux can start other processes, and these are referred to as _child processes_. `pstree` will show all processes and their children in a tree diagram.
+A helpful tool for commands you aren't familiar with is `man` - these are "manual pages" (also referred to as manpages), and they provide explanation of the commands in Linux and all of the different things you can do with them. Once you're looking at a manpage, you can use the arrow keys on your keyboard to scroll through the document, and press the `Q` key on your keyboard to exit. A large number of commands also offer the `command --help` option, which gives a less detailed but helpful nonetheless output.
+
+`top` is a command that provides your terminal interface with a constantly updating list of programs, organised from most demanding to least demanding, along with providing a summary of various system facts. It will tell you: 
+The time, the system uptime, how many users are logged in (Try the `who` command to see who is currently logged onto the system), the amount of tasks and whether they're running, idle or otherwise; It will also give you a listed breakdown of the processes that are using the most CPU power which can be helpful for identifying troublesome programs that are slowing down your system. You can press `Shift + F` to sort by lots of different metrics like RAM usage, uptime and so on, then select a metric, press enter, and then escape. You can exit the program by pressing 'Q' 
```

Comment:
this is a little bit wordy, again probably best to simplify this in the interest of making this worksheet accessible to beginners